src/streamsync/core.py

```diff
@@ -909,15 +909,14 @@ def _transform_change_page(self, ev) -> Optional[int]:
             return None
 
 
-
 class Evaluator:
 
     """
     Evaluates templates and expressions in the backend.
     It allows for the sanitisation of frontend inputs.
     """
 
-    template_regex = re.compile(r"[\\]?@{([\w\s.\[\]]*)}")
+    template_regex = re.compile(r"[\\]?@{([^{]*)}")
```

Comment:
`r"[\\]?@{([^{]*)}"`

`[\\]`: matches a single character present in the list (`\\` matches the character `\` literally)
`@{`: matches the characters `@{` literally
Capturing Group `([^{]*)`: matches a single character not present in the list (excludes `{`)
`*`: matches the previous token between zero and unlimited times, as many times as possible, giving back as needed (greedy)
`}`: matches the character `}` literally